exporters/autoexport/spans.go

```diff
@@ -18,52 +18,23 @@ import (
 	"context"
 	"os"
 
+	"go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc"
+	"go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracehttp"
 	"go.opentelemetry.io/otel/sdk/trace"
 )
 
-const (
-	otelTracesExportersEnvKey = "OTEL_TRACES_EXPORTER"
-)
-
-type config struct {
-	fallbackExporter trace.SpanExporter
-}
-
-func newConfig(ctx context.Context, opts ...Option) (config, error) {
-	cfg := config{}
-	for _, opt := range opts {
-		cfg = opt.apply(cfg)
-	}
-
-	// if no fallback exporter is configured, use otlp exporter
-	if cfg.fallbackExporter == nil {
-		exp, err := spanExporter(context.Background(), "otlp")
-		if err != nil {
-			return cfg, err
-		}
-		cfg.fallbackExporter = exp
-	}
-	return cfg, nil
-}
+// SpanOption applies an autoexport configuration option.
+type SpanOption = option[trace.SpanExporter]
```

Comment:
Should this be `TraceOption`?

I know we have the `SpanExporter`, but I wonder if this needs to match that or the corollary to metric: trace.

Reply:
We can address it in a separate PR if needed.

Personally, I was thinking about renaming to `SpanExporterOption`.

Let's track it in a separate issue.

Reply:
https://github.com/open-telemetry/opentelemetry-go-contrib/issues/4474